packages/mirinae/src/data-display/field-title/PFieldTitle.vue

```diff
@@ -1,18 +1,29 @@
 <template>
-    <span class="p-field-title"
-          v-on="$listeners"
+    <div class="p-field-title"
+         v-on="$listeners"
     >
-        <span class="title">
-            <slot>
-                {{ props.label }}
-            </slot>
-        </span>
-        <span v-if="props.description"
-              class="description"
-        >
-            {{ props.description }}
-        </span>
-    </span>
+        <div class="title-wrapper">
+            <slot name="left"
+                  class="left-slot"
+            />
+            <span class="title"
+                  :class="[fontSize, fontWeight, color]"
+            >
+                <slot>
+                    {{ props.label }}
+                </slot>
+            </span>
+            <slot name="right"
+                  class="right-slot"
+            />
+        </div>
+        <div v-if="props.description">
+            <span class="description">
+                {{ props.description }}
+            </span>
+        </div>
+        <slot name="button" />
```

Comment:
question: Why is the slot named Button?

Reply:
According to the design, there is a separate button slot.
![image](https://github.com/cloudforet-io/console/assets/83805167/be44d00f-1c88-4b3f-8425-d0e6549d4abe)

Reply:
@skdud4659 Oh... I don't think it's a space where button goes in, but I was wondering why button was used.

Reply:
I went to see the figma.
![image](https://github.com/cloudforet-io/console/assets/50662170/c31293c5-8c7a-4133-925e-9a9cddb2d283)
I think bottom is right, not button..!